mkdocs.yml

```diff
@@ -1,20 +1,95 @@
+      - Hands-on: 'bestpractices/recommendedcoursecycles/scheduled/handson.md'
+      - Workshop: 'bestpractices/recommendedcoursecycles/scheduled/workshop.md'
+      - Nutshell: 'bestpractices/recommendedcoursecycles/scheduled/nutshell.md'
+    - 'Self-paced': 'bestpractices/recommendedcoursecycles/selfpaced.md'
+    - 'Blended': 'bestpractices/recommendedcoursecycles/blended.md'
+  - Prepare Material: 'bestpractices/preparematerial.md'
+  - Quiz Design: 'bestpractices/quizdesign.md'
+  - Community Management: 'bestpractices/communitymanagement.md'
+  - End of Course: 'bestpractices/endofcourse.md'
+  - Checklist: 'bestpractices/checklist.md'
+  - Project Management Gantt Chart: 'bestpractices/projectmanagement.md'
+- Course Administration:
+  - Setup Course Properties: 'courseadministration/courseproperties.md'
+  - Grant Permissions: 'courseadministration/permissions.md'
+  - Add Content: 
+    - Sections: 'courseadministration/addcontent/sections.md'
+    - Items: 
+      - Quiz: 'courseadministration/addcontent/items/quiz.md' 
+      - Video: 'courseadministration/addcontent/items/video.md' 
+      - Text  : 'courseadministration/addcontent/items/text.md' 
+      - LTI: 'courseadministration/addcontent/items/lti.md' 
+      - Peer Assessment: 'courseadministration/addcontent/items/peerassessment.md'
+  - Setup Certificate Templates: 'courseadministration/certificatetemplates.md'
+  - Setup Learning Objectives: 'courseadministration/learningobjectives.md'
+  - Quiz Management: 
+    - Grant Additional Attempts: 'courseadministration/quizmanagement/additionalattempts.md'
+    - Regrading: 'courseadministration//quizmanagement/regrading.md'  
+  - Peer Assessment Management: 
+    - Grant Additional Attempts: 'courseadministration/peerassessmentmanagement/additionalattempts.md'
+    - Conflicts: 'courseadministration/peerassessmentmanagement/conflicts.md'
+    - Regrading: 'courseadministration/peerassessmentmanagement/regrading.md'
+- Platform Administration:
+  - Create Course: 'platformadministration/createcourse.md'
+  - Static Pages: 'platformadministration/staticpages.md'
+- Glossary: 'glossary.md'
+  
+
+
+
 - Teacher Guidelines:
-  - 'Getting Started: Designing your Course': '01_getting-started.md'
-  - 'Curriculum Design': '02_curriculum-design.md'
-  - 'Recommended Course Cycle': '03_course-cycle-schedule.md'
-  - 'Setting up the course properties': '04_course-settings.md'
-  - 'Granting permissions to users': '04a_course-permissions.md'
-  - 'Adding your content to the platform': '05_adding-content.md'
-  - 'Quiz Design': '06-quiz-design.md'
-  - 'Adding quizzes and questions': '07_adding-quizzes-and-questions.md'
-  - 'Quiz Submissions': '08_quiz-submissions.md'
-  - 'Peer Assessment': '10_peer-assessment.md'
-  - 'LTI': '11_lti.md'
-  - 'Gamification': '14_gamification.md'
-  - 'Static Pages': '15_static_pages.md'
   - 'Community Management': '19_community-management.md'
```

Comment:
A relative path to '19_community-management.md' is included in the 'nav' configuration, which is not found in the documentation files